src/containers/sidebars/data-global-sidebar/analyze-areas-sidebar-card/index.js

```diff
@@ -52,15 +52,13 @@ export const getWarningMessages = (t, locale) => ({
     description: (size) => (
       <span>
         {t(
-          'The maximum size for on the fly area analysis is {number} km{sup}.',
+          'The maximum size for on the fly area analysis is {number} km.',
           {
             number: getLocaleNumber(HIGHER_AREA_SIZE_LIMIT, locale),
-            sup: <sup>2</sup>,
           }
         )}{' '}
-        {t('The area that you are trying to analyze has {number} km{sup}.', {
+        {t('The area that you are trying to analyze has {number} km.', {
```

Comment:
I don't understand why are we removing all the ² (squares) on so many places. The units are still area so its km². Is there a reason?

Reply:
Was a request from Alex and Tamara to remove them